### PR TITLE
Move Parse_Science_Vector from INI to ScienceStore

### PR DESCRIPTION
**Pull request by duncanspumpkin**
This has been reworked and is now a move from INI to ScienceStore.

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
Suggest removing INI::Scan_Science as well since I suspect almost all users will be from the science store.

**Comment by jonwil** [comment]
This already exists as INI::Parse_Science_Vector so this PR is un-necessary.

**Comment by duncanspumpkin** [comment]
Suggest we remove the INI::Parse_Science_Vector and replace it with one in Science.cpp to make this not happen again.

File: src/game/common/rts/science.cpp [hunk 1]
```diff
@@ -68,3 +68,19 @@ ScienceType ScienceStore::Lookup_Science(const char *name)
...
+    auto &science_vec = *static_cast<std::vector<ScienceType> *>(store);
+    science_vec.clear();
+
+    for (Utf8String name = ini->Get_Next_Token_Or_Null(); name.Is_Not_Empty(); name = ini->Get_Next_Token_Or_Null()) {
+        if (name.Is_None()) {
+            science_vec.clear();
+            break;
+        }
+
+        science_vec.push_back(g_theScienceStore->Lookup_Science(name.Str()));
```
Comment (tomsons26) [inline src/game/common/rts/science.cpp]: hm this is insert in PPC...

**Review by tomsons26 — APPROVED** [review]

**Pull request merged**